src/db/config/reddit.sql

```diff
@@ -0,0 +1,42 @@
+BEGIN;
+DROP TABLE IF EXISTS users, posts, votes, comments CASCADE;
+
+CREATE TABLE users (
+    id SERIAL PRIMARY KEY,
+    name VARCHAR(255) NOT NULL,
+    username VARCHAR(50) UNIQUE NOT NULL,
+    email VARCHAR(100) NOT NULL,
+    image VARCHAR(255) NOT NULL,
+    password VARCHAR(255) NOT NULL
+);
+
+CREATE TABLE posts (
+    id SERIAL PRIMARY KEY,
+    title VARCHAR(255) NOT NULL,
+    description TEXT ,
+    user_id INTEGER REFERENCES users(id) ON DELETE CASCADE,
+    created_at TIMESTAMP NOT NULL,
+    image VARCHAR(255) NOT NULL
+);
+
+CREATE TABLE votes (
+    post_id INTEGER REFERENCES posts(id) ON DELETE CASCADE,
+    user_id INTEGER REFERENCES users(id) ON DELETE CASCADE,
+    vote BOOLEAN NOT NULL,
+    PRIMARY KEY(post_id, user_id)
+);
+
+CREATE TABLE comments (
+    id SERIAL PRIMARY KEY,
+    user_id INT,
+    post_id INT,
+    comment TEXT NOT NULL,
+     user_id INTEGER REFERENCES users(id) ON DELETE CASCADE,
+    post_id INTEGER REFERENCES posts(id) ON DELETE CASCADE,
+    created_at TIMESTAMP NOT NULL
```

Comment:
add a default value to date.now, so no need to insert value in insert query

Reply:
done